### PR TITLE
move to mongodb

### PR DESCRIPTION
**Pull request by mshd**

### CODE REVIEW

File: apps/chordstop/prisma/schema.prisma [hunk 1]
```diff
@@ -0,0 +1,88 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "mongodb"
+  url      = env("MONGODB_URI")
```
Comment (ogroppo) [inline apps/chordstop/prisma/schema.prisma:11]: 👌

**Review by ogroppo — APPROVED** [review]

File: apps/chordstop/pages/api/auth/[...nextauth].ts [hunk 2]
```diff
@@ -1,9 +1,7 @@
 import GoogleProvider from "next-auth/providers/google";
 import NextAuth from "next-auth";
 import { PrismaAdapter } from "@next-auth/prisma-adapter";
-import { PrismaClient } from "@prisma/client";
-
-const prisma = new PrismaClient();
+import { prismaClient } from "../../../prisma/prismaClient";
```
Comment (ogroppo) [inline apps/chordstop/pages/api/auth/[...nextauth].ts:4]: how can this be if it's `import { Artist } from "../prisma/generated/client";`

File: apps/wikidata-table/package.json [hunk 3]
```diff
@@ -27,7 +27,7 @@
     "node-fetch": "^3.2.4",
     "query-string": "^7.1.1",
     "react": "18.1.0",
-    "react-admin": "4.1.0",
+    "react-admin": "4.0.0",
```
Comment (ogroppo) [inline apps/wikidata-table/package.json]: why the downgrade?

Comment (mshd) [reply apps/wikidata-table/package.json]: the latest ra was broken

**Pull request merged**